File: Lab_daalgavar/lab 10/lab10.java
class Lab10 {

    public static int calculateTotalCost(int n) {
        int totalCost = 0;

        for (int i = 1; i <= n; i++) {
            if ((i & (i - 1)) == 0) { 
                totalCost += i;      
            } else {
                totalCost += 1;      
            }
        }

        return totalCost;
    }

    public static void main(String[] args) {
        int n = 10;
        int totalCost = calculateTotalCost(n);
        System.out.println("Total cost for " + n + " operations is: " + totalCost);
    }
}

File: Lab_daalgavar/lab 10/lab_10.py
import math

def calculate_total_cost(n):
    total_cost = 0
    for i in range(1, n + 1):
        if (i & (i - 1)) == 0:  
            total_cost += i  
        else:
            total_cost += 1  
    
    return total_cost

n = 10
total_cost = calculate_total_cost(n)
print("Total cost for", n, "operations is:", total_cost)
